build.gradle.kts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2001-2023 Geert Bevin (gbevin[remove] at uwyn dot com)
+ * Copyright 2022-2023 Erik C. Thauvin (erik@thauvin.net)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins {
+    buildsrc.conventions.base
+}
+
+description = "A simple defensive library to encode/decode URL components"
+group = "net.thauvin.erik"
+version = "1.3.1-SNAPSHOT"
```

Comment:
The `buildsrc.conventions.base` convention plugin will copy this group and version to subprojects that apply it, so the group and version can be defined front-and-center in an easy to find place.